locales/templates/app.pot

```diff
@@ -1202,3 +1252,160 @@ msgctxt "custom_tab_copy_url_action"
 msgid "URL copied"
 msgstr ""
 
+#. Description of what a user may find in the Advanced Setting category, used
+#. for debugging
+msgctxt "preference_advanced_summary"
+msgid "Developer tools"
+msgstr ""
+
+#. Title of Advanced Setting category
+msgctxt "preference_category_advanced"
+msgid "Advanced"
+msgstr ""
+
+#. Preference to enable remote debugging of the app via USB or Wi-Fi
+msgctxt "preference_remote_debugging"
+msgid "Remote debugging via USB/Wi-Fi"
+msgstr ""
+
+#. Title for the fingerprint authentication dialog box that is shown to the
+#. user when opening the app.      %1$s is replaced with the app name
+#, c-format
+msgctxt "biometric_auth_title"
+msgid "Unlock %1$s"
+msgstr ""
+
+#. Description for the fingerprint authentication dialog box that is shown to
+#. the user when opening the app
+msgctxt "biometric_auth_description"
+msgid "Touch the fingerprint sensor to continue."
+msgstr ""
+
+#. New session button for the fingerprint authentication dialog box that is
+#. shown to the user when opening the app.      When pressed, the user's
+#. previous session is cleared, and authentication is not required.
+msgctxt "biometric_auth_new_session"
+msgid "New Session"
+msgstr ""
+
+#. Description for the fingerprint authentication image. Used for
+#. accessibility purposes.
+msgctxt "biometric_auth_image_description"
+msgid "Fingerprint icon"
+msgstr ""
+
+#. Error text displayed when the user's fingerprint cannot be recognized upon
+#. re-entering the app.      This is displayed during the authentication flow
+#. as the app launches.
+msgctxt "biometric_auth_not_recognized_error"
+msgid "Fingerprint not recognized. Try again."
+msgstr ""
+
+#. Error text displayed when the user's fingerprint moved too quickly upon
+#. re-entering the app     This is displayed during the authentication flow as
+#. the app launches.
+msgctxt "biometric_auth_moved_too_quickly"
+msgid "Finger moved too fast. Try again."
+msgstr ""
+
+#. Title for the prompt to enable search suggestions
+msgctxt "enable_search_suggestion_title"
+msgid "Would you like to turn on search suggestions?"
+msgstr ""
+
+#. Description for the prompt to enable search suggestions         %1$s will
+#. be replaced by the app name (e.g. Firefox Focus)         %2$s will be
+#. replaced with "Learn more" which will be a tappable link         using the
+#. string: "enable_search_suggestion_subtitle_learnmore"
+#, c-format
+msgctxt "enable_search_suggestion_subtitle"
+msgid ""
+"To get search suggestions, %1$s needs to send what you type in the "
+"address bar to the search engine you've chosen. %2$s."
```

Comment:
Sorry didn't spot this the first time. As per localization guidelines, needs to be a proper apostrophe:  ’ (smart quotes). It's ok not to use them in the localization notes, but not in the actual strings.